pipelines/src/main/scala/dagr/pipelines/DownsampleAndCallSomaticVariants.scala

```diff
@@ -26,18 +26,17 @@ package dagr.pipelines
 import java.text.DecimalFormat
 
 import _root_.picard.analysis.directed.HsMetrics
-import dagr.core.cmdline.Pipelines
+import com.fulcrumgenomics.commons.CommonsDef._
+import com.fulcrumgenomics.commons.io.Io
 import com.fulcrumgenomics.sopt._
+import dagr.core.cmdline.Pipelines
 import dagr.core.execsystem.{Cores, Memory}
-import dagr.tasks.DagrDef
-import DagrDef._
 import dagr.core.tasksystem.{Linker, NoOpInJvmTask, Pipeline, SimpleInJvmTask}
-import com.fulcrumgenomics.commons.io.Io
+import dagr.tasks.DagrDef
```

Comment:
```suggestion
import dagr.tasks.DagrDef._
```
so you don't have the ugly `DagrDef.PathTo...` below

Reply:
Argh.  Thanks for catching this.  I already have `CommonsDef._` imported - this was intellij being "helpful".